lib/msf/core/auxiliary/wmap_scan_generic.rb

```diff
@@ -0,0 +1,10 @@
+
+module Msf
+  module Auxiliary::WmapScanGeneric
+    include Auxiliary::WmapModule
+
+    def wmap_type
+      :wmap_generic
+    end
+  end
+end
```

Comment:
nab: It might be nice to fix up missing trailing whitespace/remove the extra whitespace added to the top of the file :+1: